docs/development/release.rst

```diff
@@ -98,16 +143,15 @@ Steps for the day to announce the release:
     * https://groups.google.com/forum/#!forum/astropy-dev
     * CTAO AS WG list (cta-wg-as@cta-observatory.org)
     * hess-forum list (hess-forum@lsw.uni-heidelberg.de)
-#. Make sure the release milestone and issue is closed on GitHub
+#. Make sure the release milestone and issue is closed on GitHub.
 #. Update these release notes with any useful infos / steps that you learned
    while making the release (ideally try to script / automate the task or check,
    e.g. as a ``make release-check-xyz`` target).
-#. Update version number in Binder ``Dockerfile`` in
-   `gammapy-webpage repo <https://github.com/gammapy/gammapy-webpage>`__ master branch
-   and tag the release for Binder.
+#. Update the version numbers in `gammapy-webpage repo <https://github.com/gammapy/gammapy-webpage>`__
+   ``master`` branch to allow the Binder ``Dockerfile`` to be updated.
```

Comment:
Do we still wish to support this?
Note: it is not working for v1.2 or v1.3 currently

Reply:
See #5623.